src/components/CookieBanner.tsx

```
@@ -0,0 +1,62 @@
+  // 既に同意/拒否済みの場合は表示しない
+  if (hasConsent !== null) {
+    return null;
+  }
+
+  return (
+    <div 
+      className="fixed bottom-0 left-0 right-0 z-50 border-t bg-background/95 backdrop-blur supports-[backdrop-filter]:bg-background/60"
+      role="region"
+      aria-live="polite"
+      aria-label="Cookie同意"
+      aria-describedby="cookie-banner-description"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)